tests/test_sync_job_runner.py

```diff
@@ -88,6 +88,7 @@ def create_runner(
     index_name=SEARCH_INDEX_NAME,
     sync_cursor=SYNC_CURSOR,
     connector=None,
+    service_config=None,
```

Comment:
```suggestion
    service_config={},
```

Does this work? If so, you could remove the assigning on L117.

Reply:
Ohohohoh, this is a very bad practice :D

Reply:
Linking an example: https://www.pullrequest.com/blog/python-pitfalls-the-perils-of-using-lists-and-dicts-as-default-arguments/

Reply:
I didn't know D: